.jenkins

```diff
@@ -91,11 +97,12 @@ pipeline {
                                 -DKokkos_ENABLE_BENCHMARKS=ON \
                                 -DKokkos_ENABLE_HIP=ON \
                               .. && \
-                              make -j8 && ctest --verbose'''
+                              make -j8 && ctest --no-compress-output -T Test --verbose'''
```

Comment:
Did you look at the size of these XML files that are produced as compared to the size of the logs that are uploaded to the server?
Should we be worried about disk space?

Reply:
The `xml` files are between 56 KiB and 128 KiB per report:
- https://cloud.cees.ornl.gov/jenkins-ci/job/Kokkos/job/PR-7470/9/execution/node/179/ws/build/Testing/20241022-1416/ 56.07 KiB
- https://cloud.cees.ornl.gov/jenkins-ci/job/Kokkos/job/PR-7470/9/execution/node/48/ws/build/Testing/20241022-1330/ 104.55 KiB
- https://cloud.cees.ornl.gov/jenkins-ci/job/Kokkos/job/PR-7470/9/execution/node/180/ws/build/Testing/20241022-1511/ 127.70 KiB
- https://cloud.cees.ornl.gov/jenkins-ci/job/Kokkos/job/PR-7470/9/execution/node/183/ws/build/Testing/20241022-1423/ 117.89 KiB
- https://cloud.cees.ornl.gov/jenkins-ci/job/Kokkos/job/PR-7470/9/execution/node/184/ws/build/Testing/20241022-1404/ 111.32 KiB
- https://cloud.cees.ornl.gov/jenkins-ci/job/Kokkos/job/PR-7470/9/execution/node/185/ws/build/Testing/20241022-1435/ 113.52 KiB

That's negligible compared with the size of the build directories, KokkosCore_UnitTest_Cuda alone is 154.53 MiB in https://cloud.cees.ornl.gov/jenkins-ci/job/Kokkos/job/PR-7470/9/execution/node/186/ws/build/core/unit_test.

The size of the whole test output for that build is 4.3 MiB

Reply:
> negligible compared with the size of the build directories

Build directories are not uploaded to the server.  I suspect these xml files are.

Reply:
So we are talking about a maximum of 128*10 KiB or so in `xml` files which is around a quarter of the log file.

Reply:
Should we trade --verbose for --output-on-failure?
One downside is we would not be able to follow progress on a single test :/
What do you think?